planner/core/planbuilder.go

```diff
@@ -1042,6 +1027,32 @@ func (b *PlanBuilder) buildAdminCheckTable(ctx context.Context, as *ast.AdminStm
 	return p, nil
 }
 
+func (b *PlanBuilder) buildCheckIndex(ctx context.Context, dbName model.CIStr, as *ast.AdminStmt) (Plan, error) {
```

Comment:
Where do we use this function? I didn't find it.

Reply:
great, done.